### PR TITLE
Fix #343 - Remove spare % character to fix crashes on GardenActivity

### LINKED ISSUE #343: UnknownFormatConversionException Conversion = 'I'
Hey, I just cloned the repo @ 0fb46fdf495e359da178559a0ff26a32b758617c and ran the app.

I have added one plant and then pressed the back button twice, so I should have moved to the first/launcher activity. Instead, I got a `UnknownFormatConversionException`.

Attempting to run on an OnePlus5, Android Studio 3.5 Canary 3, German language.

```
2019-02-21 16:49:21.821 2436-2436/? E/ndroid.systemu: Invalid ID 0x00000000.
2019-02-21 16:49:21.833 1240-1240/com.google.samples.apps.sunflower D/AndroidRuntime: Shutting down VM
2019-02-21 16:49:21.837 1240-1240/com.google.samples.apps.sunflower E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.google.samples.apps.sunflower, PID: 1240
    java.util.UnknownFormatConversionException: Conversion = 'I'
        at java.util.Formatter$FormatSpecifier.conversion(Formatter.java:2781)
        at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2811)
        at java.util.Formatter$FormatSpecifierParser.<init>(Formatter.java:2624)
        at java.util.Formatter.parse(Formatter.java:2557)
        at java.util.Formatter.format(Formatter.java:2504)
        at java.util.Formatter.format(Formatter.java:2458)
        at java.lang.String.format(String.java:2883)
        at android.content.res.Resources.getQuantityString(Resources.java:499)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel$wateringSuffix$2.invoke(PlantAndGardenPlantingsViewModel.kt:45)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel$wateringSuffix$2.invoke(PlantAndGardenPlantingsViewModel.kt:27)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel.getWateringSuffix(Unknown Source:7)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel.<init>(PlantAndGardenPlantingsViewModel.kt:60)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter$ViewHolder.bind(GardenPlantingAdapter.kt:69)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter.onBindViewHolder(GardenPlantingAdapter.kt:49)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter.onBindViewHolder(GardenPlantingAdapter.kt:33)
        at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6781)
        at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6823)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5752)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6019)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5858)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5854)
        at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2230)
        at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1557)
        at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1517)
        at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:612)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3924)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3641)
        at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4194)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at androidx.drawerlayout.widget.DrawerLayout.onLayout(DrawerLayout.java:1231)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
2019-02-21 16:49:21.838 1240-1240/com.google.samples.apps.sunflower E/AndroidRuntime:     at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at com.android.internal.policy.DecorView.onLayout(DecorView.java:754)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2989)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2502)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1633)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7786)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1004)
        at android.view.Choreographer.doCallbacks(Choreographer.java:816)
        at android.view.Choreographer.doFrame(Choreographer.java:751)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:990)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-02-21 16:49:21.839 1240-1240/com.google.samples.apps.sunflower E/AndroidRuntime: FATAL EXCEPTION: main
    Process: com.google.samples.apps.sunflower, PID: 1240
    java.util.UnknownFormatConversionException: Conversion = 'I'
        at java.util.Formatter$FormatSpecifier.conversion(Formatter.java:2781)
        at java.util.Formatter$FormatSpecifier.<init>(Formatter.java:2811)
        at java.util.Formatter$FormatSpecifierParser.<init>(Formatter.java:2624)
        at java.util.Formatter.parse(Formatter.java:2557)
        at java.util.Formatter.format(Formatter.java:2504)
        at java.util.Formatter.format(Formatter.java:2458)
        at java.lang.String.format(String.java:2883)
        at android.content.res.Resources.getQuantityString(Resources.java:499)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel$wateringSuffix$2.invoke(PlantAndGardenPlantingsViewModel.kt:45)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel$wateringSuffix$2.invoke(PlantAndGardenPlantingsViewModel.kt:27)
        at kotlin.SynchronizedLazyImpl.getValue(LazyJVM.kt:74)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel.getWateringSuffix(Unknown Source:7)
        at com.google.samples.apps.sunflower.viewmodels.PlantAndGardenPlantingsViewModel.<init>(PlantAndGardenPlantingsViewModel.kt:60)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter$ViewHolder.bind(GardenPlantingAdapter.kt:69)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter.onBindViewHolder(GardenPlantingAdapter.kt:49)
        at com.google.samples.apps.sunflower.adapters.GardenPlantingAdapter.onBindViewHolder(GardenPlantingAdapter.kt:33)
        at androidx.recyclerview.widget.RecyclerView$Adapter.onBindViewHolder(RecyclerView.java:6781)
        at androidx.recyclerview.widget.RecyclerView$Adapter.bindViewHolder(RecyclerView.java:6823)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryBindViewHolderByDeadline(RecyclerView.java:5752)
        at androidx.recyclerview.widget.RecyclerView$Recycler.tryGetViewHolderForPositionByDeadline(RecyclerView.java:6019)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5858)
        at androidx.recyclerview.widget.RecyclerView$Recycler.getViewForPosition(RecyclerView.java:5854)
        at androidx.recyclerview.widget.LinearLayoutManager$LayoutState.next(LinearLayoutManager.java:2230)
        at androidx.recyclerview.widget.LinearLayoutManager.layoutChunk(LinearLayoutManager.java:1557)
        at androidx.recyclerview.widget.LinearLayoutManager.fill(LinearLayoutManager.java:1517)
        at androidx.recyclerview.widget.LinearLayoutManager.onLayoutChildren(LinearLayoutManager.java:612)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayoutStep2(RecyclerView.java:3924)
        at androidx.recyclerview.widget.RecyclerView.dispatchLayout(RecyclerView.java:3641)
        at androidx.recyclerview.widget.RecyclerView.onLayout(RecyclerView.java:4194)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at androidx.drawerlayout.widget.DrawerLayout.onLayout(DrawerLayout.java:1231)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
2019-02-21 16:49:21.840 1240-1240/com.google.samples.apps.sunflower E/AndroidRuntime:     at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.LinearLayout.setChildFrame(LinearLayout.java:1812)
        at android.widget.LinearLayout.layoutVertical(LinearLayout.java:1656)
        at android.widget.LinearLayout.onLayout(LinearLayout.java:1565)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.widget.FrameLayout.layoutChildren(FrameLayout.java:323)
        at android.widget.FrameLayout.onLayout(FrameLayout.java:261)
        at com.android.internal.policy.DecorView.onLayout(DecorView.java:754)
        at android.view.View.layout(View.java:20831)
        at android.view.ViewGroup.layout(ViewGroup.java:6203)
        at android.view.ViewRootImpl.performLayout(ViewRootImpl.java:2989)
        at android.view.ViewRootImpl.performTraversals(ViewRootImpl.java:2502)
        at android.view.ViewRootImpl.doTraversal(ViewRootImpl.java:1633)
        at android.view.ViewRootImpl$TraversalRunnable.run(ViewRootImpl.java:7786)
        at android.view.Choreographer$CallbackRecord.run(Choreographer.java:1004)
        at android.view.Choreographer.doCallbacks(Choreographer.java:816)
        at android.view.Choreographer.doFrame(Choreographer.java:751)
        at android.view.Choreographer$FrameDisplayEventReceiver.run(Choreographer.java:990)
        at android.os.Handler.handleCallback(Handler.java:873)
        at android.os.Handler.dispatchMessage(Handler.java:99)
        at android.os.Looper.loop(Looper.java:193)
        at android.app.ActivityThread.main(ActivityThread.java:6863)
        at java.lang.reflect.Method.invoke(Native Method)
        at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:537)
        at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:858)
2019-02-21 16:49:21.847 1240-1361/com.google.samples.apps.sunflower D/OSTracker: OS Event: crash
2019-02-21 16:49:21.851 1364-8518/? W/ActivityManager:   Force finishing activity com.google.samples.apps.sunflower/.GardenActivity
```

### PR DESCRIPTION
**Pull request by carstenhag**

### CODE REVIEW

File: app/src/main/res/values-de/strings.xml [hunk 1]
```diff
@@ -37,7 +37,7 @@
     <string name="watering_next_prefix">Zuletzt bewässert am %s</string>
     <plurals name="watering_next_suffix">
         <item quantity="one">Morgen wässern</item>
-        <item quantity="other">%In %d Tagen wässern</item>
+        <item quantity="other">In %d Tagen wässern</item>
```
Comment (XinyueZ) [inline app/src/main/res/values-de/strings.xml]: Thanks

**Comment by nic0lette** [comment]
Thank you for the PR, @carstenhag ! In order to accept it, however, we need Googlebot to be happy =( After that I'll try to merge it as soon as I can :)

**Comment by carstenhag** [comment]
@googlebot I signed it!

**Comment by nic0lette** [comment]
Villen Dank!

**Pull request merged**